---
layout: page
title: Research
image:
  feature: sunset.jpg
  credit: Gabriel Ng (UC Davis, BML)
  creditlink: /
comments: false
modified: 2015-11-30
---

*Rapid and Local Adaptation to Environmental Pollutants in Killifish*
Estuaries are among the most productive ecosystems in the world. Due to their proximity to coastal human populations, they are also among the most polluted. Fish are very sensitive to common classes of organic pollutants during early development life-stages, and in some estuaries, heavy and persistent sediment contamination has lead to the near complete extirpation of the fish normally found there. However, killifish (Fundulus heteroclitus) along the Atlantic coast have re-colonized these environments through the evolution of protection from their embryotoxicity. These industrial pollutants appear to have provided a recent and sharp selective force in separate heavily polluted locations, indicating a unique and independently evolved instances of protection. Interestingly, the pollutant profiles differ at each site and are characterized by different but overlapping mechanisms of embryotoxicity. When brought into the lab, killifish are cross-resistant to the dominant contaminant found at both sites. This system provides an excellent opportunity to study rapid evolution and further our understanding of the embryotoxicity of these classes of pollutants. Currently, I am collaborating with the USEPA Atlantic Ecology Division (Bryan Clark and Diane Nacci) to determine the genomic regions that contribute to protection in 4 separate populations of killifish. The EPA generated pollution resistant and sensitive hybrids that were assessed for their level of protection. I am using RAD-Seq and quantitative genomics to map the protective loci to the recently established killifish genome. With this information, we can determine if the same or different genomic regions are responsible for protection in the various populations and whether the different pollutants led to different evolutionary strategies.         

**Osmoregulatory Plasticity in Fish and Coastal Contaminants**
Fish actively maintain physiological homeostasis (osmotic pressure) by regulating the concentration ions in their blood plasma.The failure to do so results in chronic, systemic stress or even death. Some fish posess osmoregulatory plasticity, or the ability to change the regulation of these ions in response to environmental variation in their availibility. Fish that live in salinity variable environments such as estuaries, or those that migrate between freshwater and marine environments, must possess a high degree of osmoregulatory plasticity to concentrate or dilute plasma ions to maintain homeostasis. Salmon, for example, demonstrate a high degree of osmoregulatory plasticity by dramatically reconstructing their gill epithelium and activating physiological pathways when transitioning between freshwater and marine environments.  

Nano-materials are becoming more prevalent in coastal and estuary environments, and appear to inhibit many of the key pathways that enable morphological and physiological osmorgegulatory plasticity in fish. This is especially concerning for those species of fish that either live in contaminated estuaries, or those that pass through contaminanted environments during migration. I am attempting to characterize the impairment of nano-materials, specifically Nano Copper Oxide, on an estuary fish fundulus heteroclitus (Atlantic Killifish). Killifish have a well characterized response to environmental salinity and serve as a good model for understanding osmoregulatory function, impairment, and extreme plasticity in fish. This work is in collaboration with [Dr. Gary Cherr](http://bml.ucdavis.edu/research/faculty/gary-cherr/) at [Bodega Marine Laboratory](http://bml.ucdavis.edu/) and his participation in the [UC Center for Environmental Implications of Nanotechnology.](http://www.cein.ucla.edu/) 

**Using High-Confidence Local Genome Assembly to Identify Errors in Linkage Mapping**
Many quatitative and population genomic analysis with non-model organisms requires the development of a well assembled reference genome. However, genome assemblies are limited by the ability of short reads to accuratley assemble regions of low complexity. This results in a number high-confidence local assemblies with an unknown relationships or chromosomal locations. Meiotic mapping utilizes linkage between loci to orient the well assembled regions to one another and assigns each to a chromosomal location. However, mapping algorithms tend to treat loci independently, and when confounded with high rates of genotyping error, can lead to poor genome assemblies. I am using the high-evidence based local assembly of Fundulus heteroclitus reduce the missing and errorneous genotypes that can arise from next-gen sequencing and restriction associated digest (RAD tag). 






   



